001-create-table-and-load-data.sql

```diff
@@ -0,0 +1,15 @@
+DROP TABLE IF EXISTS live;
+
+CREATE TABLE live (
+  id int unsigned AUTO_INCREMENT,
+  schedule DATETIME NOT NULL,
+  name VARCHAR(100) NOT NULL,
+  location VARCHAR(100) NOT NULL,
+    PRIMARY KEY(id)
+);
+
+INSERT INTO live (schedule ,name ,location) VALUES ('2024-5-9 19:00:00' ,'Yngwie J.Malmsteen' ,'zepp namba');

```

Comment:
```suggestion
INSERT INTO live (schedule, name, location) VALUES ('2024-5-9 19:00:00', 'Yngwie J.Malmsteen', 'zepp namba');

```
絶対の決まりはないものの英語ではカンマの後ろに半角スペースが自然です。

Reply:
カンマの後ろに半角スペースで修正しました。
ご確認をお願いします。https://github.com/kttsu/learning_task_10/pull/1#discussion_r1613632352